ICAFIX/hcp_fix_multi_run

```diff
@@ -126,6 +130,9 @@ opts_AddOptional '--processing-mode' 'ProcessingMode' '"HCPStyleData" (default)
 
 opts_AddOptional '--concatenate-only' 'ConcatOnlyStr' 'TRUE or FALSE' "When set to TRUE the script stops after the concatination step. e.g., for use in experimental alternative multi-run denoising" 'FALSE'
 
+opts_AddOptional '--ica-method' 'ICAmethod' 'MELODIC or ICASSO' "Use single-pass MELODIC (default) or hundred-pass ICASSO consensus method for ICA" 'MELODIC'
```

Comment:
I don't see any sanity checking of this value, so if someone typed `--ica-method=ICASOO`, it looks like the script will neither error nor use either of these options.  My usual syntax for things like this is case:
```
case "$ICAmethod" in
    (foo|bar)
        #fine, don't error
        ;;
    (*)
        log_Err_Abort "not fine"
        ;;
esac
```
In general, I prefer case to elif when it would be repeatedly testing the same variable.